operators/src/processing/point_in_polygon.rs

```diff
@@ -267,14 +278,18 @@ where
 #[cfg(test)]
 mod tests {
 
+    use std::str::FromStr;
+
```

Comment:
if you change the assertion to matches, could you at the same time put this thing under `use super::*;`?